### ISSUE TITLE
On the issue of PNSR being considerably reduced.

### ISSUE THREAD

**Issue by hiroesta** [megvii-research/WACV2024-SAFA#8]
Hello author.

The following codes and options were used for the training. (Code rewritten to work with that option, otherwise unchanged)
`python3 -m torch.distributed.launch --nproc_per_node=1 train.py --world_size=1 --batch-size=4`
The result was a pnsr of about 10 at 4k steps.
There is a considerable discrepancy with the results in the paper, do you know why?

**Comment by hiroesta** [comment]
I used adobe240fps dataset for this training.

**Comment by hzwer** [comment]
The learning rate for this type of work is usually set at a relatively high threshold. If using a single card (implying a smaller batch size), it is often necessary to reduce the learning rate. You need to monitor the loss function curve to ensure that the training does not collapse due to an excessively high learning rate.

**Comment by kbui1993** [comment]
I also have been trying to train the model from scratch. In train.py, the lowres wasn't normalized (i.e., wasn't divided by 255) before being passed into the model. Like @hiroesta , I was getting psnr of around 10 at the end of the first epoch. However, normalizing fixes it.

**Comment by hyx1203** [comment]
> I also have been trying to train the model from scratch. In train.py, the lowres wasn't normalized (i.e., wasn't divided by 255) before being passed into the model. Like [@hiroesta](https://github.com/hiroesta) , I was getting psnr of around 10 at the end of the first epoch. However, normalizing fixes it.
Hello, do you have the test code here? I would be very grateful if you could provide it!
